### PR TITLE
TESLA-1064: Add GTM to edX 

### PR DESCRIPTION
**Pull request by lukyfebrianto**
**Description:** Add GTM to edX

**JIRA:** https://liv-it.atlassian.net/browse/TESLA-1064

### CODE REVIEW

File: themes/courses.labster.com/lms/templates/main.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
+## coding=utf-8
+
```
Comment (auraz) [inline themes/courses.labster.com/lms/templates/main.html:2]: Where this template is used? Is it plugged in automatically?

**Review by leha2000 — APPROVED** [review]

File: themes/courses.labster.com/lms/templates/main.html [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+## This is the main Mako template that all page templates should include.
```
Comment (akoidan) [inline themes/courses.labster.com/lms/templates/main.html:6]: Add up a comment that this file was copied from that place, and it's exactly the same.

And most important highlight a line in this file that makes it different from the original one.

Comment (akoidan) [reply themes/courses.labster.com/lms/templates/main.html:6]:
```
1 ### This is a copy of #file
2 The only difference is in line #10, 
...
8
9
10 # this is the only line different from original file
11 GOOGLE_TAG 
12 ....
```

**Review by akoidan — CHANGES_REQUESTED** [review]

File: themes/courses.labster.com/lms/templates/main.html (hunk 2 above)
Comment (auraz) [reply themes/courses.labster.com/lms/templates/main.html:6]: That would be very helpful!

Comment (lukyfebrianto) [reply themes/courses.labster.com/lms/templates/main.html:6]: added

Comment (auraz) [reply themes/courses.labster.com/lms/templates/main.html:6]: Thank you, approved!

**Review by auraz — APPROVED** [review]

**Review by akoidan — APPROVED** [review]

**Pull request merged**